engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -256,6 +258,11 @@ private void initRenderGraph() {
         renderGraph.addNode(toneMappingNode, "toneMappingNode");
         renderGraph.addNode(bloomPassesNode, "bloomPassesNode");
         renderGraph.addNode(blurPassesNode, "blurPassesNode");
+        if (renderingConfig.isVrSupport()) {
```

Comment:
At this stage nodes are enabled or disabled via the ConditionDependentNode.requireCondition() method. So, no need to have an IF block here. And there are no performance consequences as a disabled node has no footprint in the rendering tasklist, which is what gets executed to render every frame. See the LightShaftsNode for a simple example on how to disable a node monitoring a rendering config property.

Eventually nodes will need to add or remove themselves from the node list (eventually node graph) depending on config parameters and any other criteria they wish to establish. Something for you to keep in mind for later.